### DISCUSSION TITLE
Search Button Old and New

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2919] [Ideas]
The search button that is in Explorer Patcher is the old one (I think)
![Old Windows 10 Search Button](https://github.com/valinet/ExplorerPatcher/assets/152800265/8ec858d3-6b62-455b-8897-519f00f0aae7)
and this one is the new one for Windows 10
![New Windows 10 Search Button](https://github.com/valinet/ExplorerPatcher/assets/152800265/f75ee77d-5804-4c94-b057-3266d619763c)
(these pictures are from native Windows 10)

I like both but the old one that is in Explorer Patcher is low resolution, it looks pixelated (from Windows 11 with Explorer Patcher)
<img width="82" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/4c0aaf81-1770-468d-9797-839d2277762a">
you can see it's pixelated. Is there any way to increase the old Search Button's resolution and add the new Search Button that is in Windows 10?

**Comment by pyrates999** [comment]
did you install winaero tweaker?  If you did, install it again if you uninstalled it and reset its settings there.

It has a setting that if used, can cause icons to be blurry in the system tray.  It might be causing the search icon to be blurry.

**Reply by TimMarvelHero** [reply]
I did not install Winaero.

**Reply by TimMarvelHero** [reply]
But still thanks.

**Comment by haydengriz** [comment]
A small detail, but it would be nice to have this as an option anyway. Also I think this is a duplicate(?)

**Reply by TimMarvelHero** [reply]
Yes. 

What duplicate?

**Reply by haydengriz** [reply]
I think someone already commented about this suggestion on GitHub before you

**Reply by TimMarvelHero** [reply]
about what?

**Reply by haydengriz** [reply]
dude, the search button? hello?

**Reply by TimMarvelHero** [reply]
I was talking about the low resolution search button and the new search button in Windows 10, I don't know which one you're talking about lol

**Comment by Amrsatrio** [comment]
I will consider about it in the reimplemented taskbar that should be compatible with 22621 and up.

**Reply by haydengriz** [reply]
So the old one will still be an option too?

**Reply by TimMarvelHero** [reply]
I think the new one will be used as the search button with a label in Windows, the old one can be used as the normal icon search button since there are 4 options to customize your search in Windows, can be hidden, can be an icon button, can be an icon with a label, and a full search box.

**Comment by Kichura** [comment]
The new search icon will not be restored as it looks out of place and doesn't align with Windows 10's actual design language.
